src/app/books.ts

```diff
@@ -2433,12 +2421,208 @@ export const BOOKS : Book[] = [
 			"fimfic", "hardcover", "custom"
 		]
 	}
+	,{
+		"id": 84,
+		"title": "Equestrian Stories",
+		"edition": "1",
+		"img": "https://c1.iggcdn.com/indiegogo-media-prod-cld/image/upload/c_fill,w_695,g_auto,q_auto,dpr_1.0,f_auto,h_460/llnzg3uv6kl7vf9npf4r",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": "2019/04/30",
+		"authors":[
+			{
+				"name": "Nonexistant Publications",
+				"url": "https://www.fimfiction.net/user/129042/Avonder"
```

Comment:
This should be linked to the story authors. I do not think a single person authored all of these stories.